### ISSUE TITLE
00:00 after game end when switching "restart from zero"

### ISSUE THREAD

**Issue by SMD2023** [msaltnet/handball-stopwatch#26]
Today the game was in an different city and the main timer reset to 00:00 (restart from Zero) after break. (moste time it continues)
I don't know that before and try to switch in break. 
This doesn't work. It would be great if it is possible to switch the mode in break time but it is okey if it is not possible. 

But 
I was wondering that the main-time is set to 00:00 at end of the game. Rest of main display is freezing. Only timer is set to 00:00. 
Only a minimal bug and not relevant for main function. I would like to report it because I recognized it today.

**Comment by msaltnet** [comment]
> I don't know that before and try to switch in break.
This doesn't work. It would be great if it is possible to switch the mode in break time but it is okey if it is not possible.

You mean user can't change restart mode in break time? Yes, it should be decided when game initialized.

> I was wondering that the main-time is set to 00:00 at end of the game. Rest of main display is freezing. Only timer is set to 00:00.

I don't understand what you said. The main timer is set to the game time at the end of the game (e.g. 20:00).

Please let me know any small bugs, anytime when it come out.

**Comment by SMD2023** [comment]
@
![IMG_20230306_161558](https://user-images.githubusercontent.com/124830764/223151653-57db41da-2aa8-4626-9491-c55d7ffbc57a.jpg)

This is a picture at end of game. 
Regularly there must be a freezed 60:00 (in this case).

But because I tried to change continue/set to zero after break the app set the maintimer to 00:00 at end of the game.

Do not know exactly why but I am sure it is because I tried to change a setting which could not be changed  😀
